From 5cec3b1701796aa834c7cc79415252c4e31045f0 Mon Sep 17 00:00:00 2001
From: Ivan Garramona <heavenandhell171@gmail.com>
Date: Wed, 13 Jan 2016 16:33:19 -0200
Subject: [PATCH 8/9] Tests should return non zero value on failure.

---
 run_tests.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/run_tests.py b/run_tests.py
index 6f009bd..5eb79e4 100755
--- a/run_tests.py
+++ b/run_tests.py
@@ -66,6 +66,7 @@ for _, doc, cases in parse_test(tests):
 
 if failures:
 	print "%d failures" % failures
+	exit(1)
 else:
 	print "PASS (%d)" % passes
 
-- 
2.24.0

